src/j1client.js

```diff
@@ -90,10 +90,21 @@ class JupiterOneClient {
     let results = [];
 
     while (!complete) {
+      // is this a good syntax for enums in JS? If so I'll move it out of the function's scope.
+      const JobStatus = {
```

Comment:
This is reasonable for creating an enum. It would be more straightforward in typescript because typescript will automatically generate this object for you.